src/components/splash/Splash.js

```diff
@@ -127,13 +127,11 @@ const styles = StyleSheet.create({
   container: {
     alignItems: 'center',
     justifyContent: 'center',
-    transform: [{ rotateY: '180deg' }],
     position: 'relative',
     backgroundColor: 'transparent',
     flex: 1,
   },
   content: {
-    transform: [{ rotateY: '180deg' }],
```

Comment:
are you sure? this doesnt change the way the background is displayed?

Reply:
@sirpy if we need to change the way the background is displayed, we need to change the .svg file. Applying the rotation style, we greatly reduce the resolution of elements inside the rotated component

Reply:
so change the svg, you cant change functionality/look and feel without finding a solution!
@serdiukov-o-nordwhale

Reply:
@sirpy we are developers not designers
We are working with the assets provided, not changing them

I have some vector editors installed could handle svg. Will try to adjust image, hope it will be easy (like just select something, apply rotation transform etc)
If I won’t have any success I’ll ask @patpedrosa to ask designer to do it

Reply:
this is not the point, i dont care how you change the svg or what solution you find, the point is that we can't just decide to change the way the design looks because it solves us a code problem

Reply:
i understand You. sorry the last week was full of PRs, and this one var very urgent so i had just skip this because we had most critical things related to the iOS app
I've asked @roma-hladilka he could adjust svg (this is possible very easy, directly in the Figma)